app/classifier/classifier.jsx

```diff
@@ -298,9 +298,11 @@ class Classifier extends React.Component {
       .then(() => {
         workflowHistory = [];
         this.setState({ annotations, showIntervention, showSummary, workflowHistory });
-        return showLastStep ? null : this.onNextSubject();
       })
       .then(onComplete)
+      .then(() => {
```

Comment:
This might be a bug in #4781. I don't think you want to call `onNextSubject` before `onComplete` so this reverses the order of those calls.